psi4/header.py

```diff
@@ -68,6 +68,10 @@ def print_header():
     J. Chem. Theory Comput. 13(7) pp 3185--3197 (2017).
     (doi: 10.1021/acs.jctc.7b00174)
 
+
+                         Additional Contributions by
+    P. Kraus, H. Kruse, M. H. Lechner, M. C. Schieber, and R. A. Shaw
```

Comment:
Really awesome guys! @PeterKraus @hokru @mhlechner @schiebermc @robashaw Really and truly thanks for the commits, they are making Psi4 a much better program.